spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.cloud.gcp.spanner.projectId=amp-demo-landing-page
```

Comment:
let's change this to `<your-project-id>`

Reply:
hahaha oh yes.

Reply:
@balopat changes made!